### PR TITLE
handle videocall invitations

### PR DESCRIPTION
**Pull request by r10s**
this pr adds sending and receiving videocall invitations as provided by the new api from https://github.com/deltachat/deltachat-core-rust/pull/1735

the call itself is done in a browser or in apps as "jitsi meet".

Screen Shot 2020-07-29 at 16 52 17 Screen Shot 2020-07-29 at 16 54 15
Screen Shot 2020-07-29 at 16 54 37 Screen Shot 2020-07-29 at 16 55 10

to use jitsi for video calls, you can scan the following qr-code:

### CODE REVIEW

File: res/drawable/ic_map_white_24dp.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#FFFFFF"
```
Comment (r10s) [inline res/drawable/ic_map_white_24dp.xml:1]: the map-icon is updated in this pr as at in the first iteration, there was a video-icon as well and the map-icon did not match standard-sizes. later, we decided to remove video-icon - but the map-icon is still nicer :)

**Comment by cyBerta** [comment]
Currently the value of the entry 'Video chat instance' doesn't update when I add "https://meet.jit.si/$ROOM"

**Comment by r10s** [comment]
@cyBerta looks to me as if not the correct core-rust-submodule is used or build, indeed.

File: src/org/thoughtcrime/securesms/ConversationFragment.java [hunk 2]
```diff
@@ -731,20 +735,6 @@ public void onItemLongClick(DcMsg messageRecord) {
...
-            if (getContext() == null) return;
-
-//      ContactUtil.selectRecipientThroughDialog(getContext(), choices, locale, recipient -> {
-//        CommunicationActions.startConversation(getContext(), recipient, null);
-//      });
-        }
-
-        @Override
-        public void onInviteSharedContactClicked(@NonNull List<Recipient> choices) {
-            if (getContext() == null) return;
```
Comment (cyBerta) [inline src/org/thoughtcrime/securesms/ConversationFragment.java:746]: :)

**Pull request merged**